h2o-py/tests/testdir_misc/pyunit_export_parquet_npe.py

```diff
@@ -0,0 +1,21 @@
+import sys
```

Comment:
Please add a R test.  You can copy the test in the issue.

Reply:
You can add this test to h2o-3/h2o-r/tests/testdir_parser after given it a meaningful name:

-------------------------------
setwd(normalizePath(dirname(
  R.utils::commandArgs(asValues = TRUE)$"f"
)))
source("../../scripts/h2o-r-test-setup.R")

test.parseParquetString<- function() {
df <- h2o.createFrame(rows = 100,
                      cols = 10,
                      string_fraction = 0.1, # create one string column
                      seed = 5,
                      seed_for_column_types = 25)
 target <- file.path(sandbox(), "createdFrame.parquet")
h2o.exportFile(data = df,
               path = target,
               format = "parquet",
               write_checksum = FALSE)
df2 <- h2o.importFile(target)
compareFrames(df, df2)
}

doTest("Test Parquet String export error.", test.parseParquetString)
---------------------------------